src/components/Launch/useDefaultLaunchPlan.ts

```diff
@@ -0,0 +1,36 @@
+import { useAPIContext } from 'components/data/apiContext';
+import { useFetchableData } from 'components/hooks';
+import { NotFoundError } from 'errors';
+import { LaunchPlan, WorkflowId } from 'models';
+
+/** Returns a fetchable for the default launch plan whose project/domain/name/version
+ * match the values in the given WorkflowId.
+ */
+export function useDefaultLaunchPlan(workflowId: WorkflowId | null = null) {
```

Comment:
Should this be organized under `hooks` somewhere? Or are you keeping ones that are intended only for specific components under that component.

Reply:
I'm trying to move away from keeping all the hooks in a common folder and instead colocating hooks in the modules that use them.